muckrock/core/storage.py

```diff
@@ -30,7 +33,14 @@ def save(self, name, content):
 
 
 class MediaRootS3BotoStorage(S3Boto3Storage):
+    """
+    S3 storage backend for user-uploaded media files. 
+    (It may or may not use the same bucket as static files.)
+    """
     file_overwrite = False
+    bucket_name = settings.AWS_MEDIA_BUCKET_NAME
+    querystring_auth = settings.AWS_MEDIA_QUERYSTRING_AUTH
+    custom_domain = settings.AWS_MEDIA_CUSTOM_DOMAIN
```

Comment:
We should double-check that these have appropriate defaults. (They should fall back to the original `AWS_STORAGE_BUCKET_NAME` etc, but worth double-checking since that might be a pain to debug if not.)

Reply:
I double checked and they look good to me?